doc/glm/meaning_difference.rst

```diff
@@ -0,0 +1,61 @@
+.. _meaning_difference:
+
+==================================================
+Difference in meanings between different toolboxes
+==================================================
+
+.. topic:: **Page summary**
+
+	* SPM has the same meaning of analysis levels, compared to nilearn's models
+		* First-level: analyze across sessions
```

Comment:
```suggestion
		* First-level: analyze across runs
```
Following BIDS convention, runs and sessions are different things, and you typically don't analyze across sessions in a first-level model.

Reply:
Agreed, great catch, thank you!!!